persistence/persistence-api/src/main/java/org/cbioportal/persistence/GenericAssayRepository.java

```diff
@@ -0,0 +1,17 @@
+package org.cbioportal.persistence;
+
+import java.util.HashMap;
+import java.util.List;
+
+import org.cbioportal.model.meta.GenericAssayMeta;
+
+public interface GenericAssayRepository {
+
+    List<GenericAssayMeta> getGenericAssayMeta(List<String> stableIds);
```

Comment:
The @Cacheable annotation should be added to these methods.  See [MolecularDataRepository](https://github.com/cBioPortal/cbioportal/blob/master/persistence/persistence-api/src/main/java/org/cbioportal/persistence/MolecularDataRepository.java) for an example.